src/views/Drawer/DrawerView.js

```diff
@@ -138,11 +139,26 @@ export default class DrawerView<T: *> extends PureComponent<void, Props, void> {
     const DrawerScreen = this.props.router.getComponentForRouteName(
       'DrawerClose'
     );
+
+    const screenNavigation = addNavigationHelpers({
+      state: this._screenNavigationProp.state,
+      dispatch: this._screenNavigationProp.dispatch,
+    });
+
+    const config = this.props.router.getScreenOptions(
+      screenNavigation,
+      'drawer'
+    );
+
     return (
       <DrawerLayout
         ref={(c: *) => {
           this._drawer = c;
         }}
+        drawerLockMode={
+          (this.props.screenProps && this.props.screenProps.lockMode) ||
+          (config && config.lockMode)
```

Comment:
Should this be `drawerLockMode` rather than `lockMode`?

Reply:
`drawerLockMode` sounds better to me; more descriptive